tests/integration/test_script_runner.py

```diff
@@ -595,63 +703,54 @@ def test_integration_tests(test_configuration, tmp_path, pytest_parsed_arguments
     _execute_integration_test(test_configuration, tmp_path)
 
 
-def _execute_integration_test(test_configuration, tmp_path):
+def _execute_integration_test(integration_test_fixture, tmp_path):
     """
     Prepare and environment and run integration tests from a list of tests.
 
-    Note that the only required parameter for a test in the matrix is
+    N ote that the only required parameter for a test in the matrix is
     `user_flow_script` and that all other parameters are optional.
     """
-    assert (
-        "user_flow_script" in test_configuration.keys()
-    ), "a `user_flow_script` is required"
     workdir = os.getcwd()
     try:
-        base_dir = test_configuration.get(
-            "base_dir", file_relative_path(__file__, "../../")
-        )
+        base_dir = file_relative_path(__file__, "../../")
```

Comment:
Base dir is always the same so we removed it from the test config?

Reply:
yes that's correct 👍🏼